openhands/server/config/openhands_config.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from openhands.server.types import AppMode, OpenhandsConfigInterface
+from openhands.utils.import_utils import import_from
+
+
+class OpenhandsOssConfig(OpenhandsConfigInterface):
+    CONFIG_PATH = os.environ.get('OPENHANDS_CONFIG_PATH', None)
+    APP_MODE = AppMode.OSS
+    POSTHOG_CLIENT_KEY = 'phc_3ESMmY9SgqEAGBB6sMGK5ayYHkeUuknH2vP6FmWH9RA'
+    GITHUB_CLIENT_ID = os.environ.get('GITHUB_APP_CLIENT_ID', '')
```

Comment:
Is this really only used for the web server, doesn't the resolver need this integration?

Reply:
Yeah resolver doesn't need this

Reply:
Sorry, can you help me understand this?
https://openhands-ai.slack.com/archives/C07V9BMGHH6/p1733259297487259

Isn't a github app the solution we go with for the resolver on forks: https://github.com/All-Hands-AI/OpenHands/issues/5050

Reply:
For sure! We run the current resolver standalone (`python -m openhands.resolver.resolver_issue`); it doesn't use the OH server so it has not concept of AppMode and doesn't need any other server attributes either

The resolver is kicked off as follows

1. [Install openhands-ai](https://github.com/All-Hands-AI/OpenHands/blob/main/.github/workflows/openhands-resolver.yml#L173)
2. [Run the resolver directly](https://github.com/All-Hands-AI/OpenHands/blob/main/.github/workflows/openhands-resolver.yml#L187)


Because it was standalone we initially had it in a [separate repo](https://github.com/All-Hands-AI/openhands-resolver) but [decided to merge](https://github.com/All-Hands-AI/OpenHands/pull/4964) it with OH proper later